main/templates/footer.html

```diff
@@ -74,3 +74,4 @@ <h3>MIT Bootcamps</h3>
     </div>
   </div>
 </footer>
+<div id="footer-notifications" class="fixed-bottom"></div>
```

Comment:
I think a better approach would be to add an empty block to `base.html` (called something like `postfooter`), then fill in that block with this element in `react.html`. As a bonus, you might be able to get rid of the DOMContentLoaded event listener if you do this

Reply:
This div needs to go below the `footer` element, otherwise the footer notifications stop just above the footer when scrollin

Reply:
It can be still be placed below the footer with the approach I suggested. I'm saying we should add a new block to `base.html`, which already defines a block for the footer.

```
# base.html
    {% block footer %}
      {% include "footer.html" %}
    {% endblock %}
    {% block postfooter %}
    {% endblock %}

# react.html
{% block content %}
<div id="app-container" class="top-level-page-body">
</div>
{% endblock %}
{% block postfooter %}
<div id="footer-notifications" class="fixed-bottom">
</div>
{% load render_bundle %}
{% render_bundle "root" %}
{% endblock %}
```